WooCommerce/Classes/ViewRelated/Products/Media/WordPressMediaLibraryPickerCoordinator.swift

```diff
@@ -4,27 +4,30 @@ import struct Yosemite.Media
 
 /// Coordinates navigation for picking media from the site's WordPress media library.
 /// `NSObject` is required for `UIAdaptivePresentationControllerDelegate` conformance.
-final class WordPressMediaLibraryImagePickerCoordinator: NSObject {
-    typealias Completion = WordPressMediaLibraryImagePickerViewController.Completion
+final class WordPressMediaLibraryPickerCoordinator: NSObject {
+    typealias Completion = WordPressMediaLibraryPickerViewController.Completion
```

Comment:
I'll keep this with the same explanation as the above response:

> It might not be as readable as keeping the raw type, but it gives the context about where the closure is returned from